Branch-SDK/Branch-SDK/BNCContentDiscoveryManager.m

```diff
@@ -40,18 +40,14 @@ @implementation BNCContentDiscoveryManager
 
 - (NSString *)spotlightIdentifierFromActivity:(NSUserActivity *)userActivity {
 #if defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 90000
-    if ([userActivity.activityType hasPrefix:BRANCH_SPOTLIGHT_PREFIX]) {
-        return userActivity.activityType;
```

Comment:
For existing integrations, I think we should fallback to `activityType` in case items were indexed with the old method. Eh?